fastavro/reader.py

```diff
@@ -54,7 +54,7 @@ def read_boolean(fo, schema):
 
     # technically 0x01 == true and 0x00 == false, but many languages will cast
```

Comment:
Maybe we'd like to be more strict here and allow only 0 and 1? Will think about it.